defaults/main.yml

```diff
@@ -67,3 +67,4 @@ opencraft_all_env_tokens: "{{ OPENCRAFT_GENERATED_ENV_TOKENS | combine( OPENCRAF
 www_data_home_dir: '/var/www'
 opencraft_root_dir: '/var/www/opencraft'
 opencraft_virtualenv_dir: '/var/www/.virtualenvs/opencraft'
+OPENCRAFT_STATIC_FILES_DIR: '{{ opencraft_root_dir }}/build/static'
```

Comment:
In some cases we use uppercase variables for defaults that are supposed to be changed by users of the role, and lower case variables for values that should not be overridden.  I'm not sure we are following this policy in this repo, but for consistency with the other directory names we should use a lower-case name here.

The static file directory is hard-coded in Ocim, so it definitely can't be changed by users in a meaningful way.